### PR TITLE
Feat-211: update validators table - styles, formatting and sorting

### PR DESCRIPTION
**Pull request by mgibson-casper**
### 🔥 Summary
- Update validators table to include formatted columns and sorting

### 📹 Video
https://user-images.githubusercontent.com/124628688/221926151-61ed02d5-111b-4bf3-aaaa-89b376f41717.mov

### 😤 Problem / Goals
- Currently the table has minimal styles and no sorting

### 🤓 Solution
-

### 🗒️ Additional Notes
- We are still waiting on UI/UX for designs on what a "neutral" column (one that is sortable, but not sorted currently) should look like. For now, we are just using a blank background icon.
- See https://github.com/casper-network/casper-blockexplorer-middleware/pull/27 for corresponding middleware implementation. Will need to merge at the same time.

### CODE REVIEW

**Review by Thistle-Strum — APPROVED** [review]

File: app/src/components/base/Table/Table.tsx [hunk 1]
```diff
@@ -60,47 +60,55 @@ export function Table<T extends unknown>({
...
+                      key={header.id}
+                      colSpan={header.colSpan}
+                      style={{ width: header.getSize() }}
+                      sortable={header.column.getCanSort()}
+                      onClick={() =>
+                        header.column.getCanSort()
+                          ? header.column.toggleSorting(
+                              header.column.getIsSorted() === 'asc',
+                            )
+                          : undefined
```
Comment (DorianKwan) [inline app/src/components/base/Table/Table.tsx:81]: NIT: Thoughts on adding a helper function on `.column` to handle this?

**Review by DorianKwan — APPROVED** [review]
LGTM!

File: app/src/components/base/Table/Table.tsx (hunk 1 above)
Comment (mgibson-casper) [reply app/src/components/base/Table/Table.tsx:81]: Ah, probably in the future. This whole table file is a bit of mess, and eventually I'll go and clean it up - it's all the tanstack stuff, which is not super readable. If you're OK, I'll just keep as in for now?

Comment (DorianKwan) [reply app/src/components/base/Table/Table.tsx:81]: Yep works for me, you've got an approval :)

**Review by valfaro123 — APPROVED** [review]
Nice work!

**Pull request merged**